internal/auth/jwt.go

```diff
@@ -64,10 +73,12 @@ func AddClaimHandler(next http.Handler, logger *zerolog.Logger) http.Handler {
 
 		telClaim, ok := claims.CustomClaims.(*TelemetryClaim)
 		if !ok {
-			logger.Error().Msg("could not cast claims to telemetyClaim")
+			logger.Error().Msg("Could not cast claims to TelemetryClaim")
 			jwtmiddleware.DefaultErrorHandler(w, r, jwtmiddleware.ErrJWTMissing)
+			return
```

Comment:
I think this return is good. I doubt one could have hit it in practice, but still.